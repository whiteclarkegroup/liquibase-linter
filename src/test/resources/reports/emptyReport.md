# Summary
* ERROR: 0
* IGNORED: 0
* PASSED: 0
* DISABLED: 0
