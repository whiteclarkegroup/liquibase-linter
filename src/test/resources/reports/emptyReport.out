Summary:
	ERROR: 0
	IGNORED: 0
	SKIPPED: 0
	PASSED: 0
	DISABLED: 0
